test/workload-checks/typical/cases/app_server/experiment.yaml

```diff
@@ -0,0 +1,70 @@
+description: >
+  Simulates a relatively busy application server on which DogStatsD metrics,
+  traces and TCP streamed logs are present on which the client user is
+  interested in OTeL but has not made a complete transition. DogStatsD, TCP logs,
+  and APM traces make up the load. We make claims about throughput, UDS packet
+  loss and memory, CPU resource consumption.
+
+teams:
+  - agent-core
+
+labels: {}
+
+checks:
+  - name: memory_usage
+    description: "Memory usage"
+    bounds:
+      series: rss_bytes
+      # The machine has 12Gb free.
+      upper_bound: 3.5Gb
+
+  - name: cpu_utilization
+    description: "CPU utilization"
+    bounds:
+      series: cpu_percentage
+      # The machine has 8 cores available.
+      upper_bound: 400
+
+  - name: apm_trace_throughput
+    description: "APM throughput"
+    bounds:
+      series: "rate(bytes_written['traces'])"
+      # Lading is configured to send 8 MB/s on each of two connections. This is
+      # a relatively high priority concern by bounding below close to the
+      # transmission rate.
+      lower_bound: 4b
+      upper_bound: 10Mb
+
+  - name: dogstatsd_throughput
+    description: "DogStatsD throughput"
+    bounds:
+      series: "rate(bytes_written['dogstatsd'])"
+      # Lading is configured to send 32 MB/s. This is a relatively high priority
+      # concern by bounding below close to the transmission rate.
+      lower_bound: 0b
```

Comment:
similar to a previous comment, lower_bound is `0 bytes` which is not "bound close to the transmission rate"

Reply:
Addressed in [b09e52b](https://github.com/DataDog/datadog-agent/pull/18814/commits/b09e52bbc04d7901437d78dd2e3934031d3f383d)